java/src/main/resources/config-file.yaml

```diff
@@ -0,0 +1 @@
+{ selector: span, transform: 'urlQueryParam(q, utm_source)' }
```

Comment:
we should remove this file and use a raw string when a yaml configuration needed.